mmeds/config.py

```diff
@@ -19,7 +19,11 @@
     ROOT = Path(mmeds.__file__).parent.resolve()
     HTML_DIR = Path(html.__file__).parent.resolve()
     STORAGE_DIR = Path(resources.__file__).parent.resolve()
-    DATABASE_DIR = Path().home() / 'mmeds_server_data'
+    # For some reason my laptop can't find Path().home()
```

Comment:
```suggestion
    # If apache is trying to access apache's home directory for `mmeds_server_data` rather than your user account's home
    # you may need to hardcode the path here
```